ISSUE_TEMPLATE.md

```diff
@@ -1,6 +1,5 @@
 > ATTENTION! Please read and follow:
-> - if this is a _question_ about how to build / test / query / deploy using Bazel, ask it on StackOverflow instead: https://stackoverflow.com/questions/tagged/bazel
-> - if this is a _discussion starter_, send it to bazel-discuss@googlegroups.com
+> - if this is a _question_ about how to build / test / query / deploy using Bazel, or a _discussion starter_, send it to bazel-discuss@googlegroups.com
```

Comment:
I wonder if we should discuss this on bazel-discuss@googlegroups.com first. I'm not a big fan of having all the random questions sent to 1,000 people, but it's true that StackOverflow is not in a great state.

If we change this, we might want to update https://www.bazel.build/support.html#get-support too.

Reply:
> I wonder if we should discuss this on bazel-discuss@googlegroups.com first. I'm not a big fan of having all the random questions sent to 1,000 people, but it's true that StackOverflow is not in a great state.

There is not much to discuss there - we are not providing support on SO, so "ask on SO" = "go away". We do monitor and reply on bazel-discuss@. This change just aligns our documentation with reality.

Reply:
https://github.com/bazelbuild/bazel-website/pull/201 for updating support.html

Reply:
It's not "go away". Answering questions is also a community activity. For example, 630 people are on the Bazel slack, they use it actively for asking/answering user questions. But we don't provide any official support there. So something doesn't need to be officially monitored to be useful.

But sure, we can do this change.